x/circuit/features/msg_authorize.feature

```diff
@@ -0,0 +1,87 @@
+Feature: MsgAuthorizeCircuitBreaker
+
+  Circuit breaker actions can be authorized:
+  - when the granter is a super-admin
+  - when the permissions are valid
+
+  Rule: the granter must be a super-admin
+    Example: granter is a super-admin
+      Given "acct1" has permission "LEVEL_SUPER_ADMIN"
+      When  attempts to grant "acct2" the permissions
+      """
+      { "level": "LEVEL_ALL_MSGS" }
+      """
+      Then expect success
+
+    Example: granter has no permissions
+      Given "acct1" has no permissions
+      When "acct1" attempts to grant "acct2" the permissions
+      """
+      { "level": "LEVEL_ALL_MSGS" }
+      """
+      Then expect an "unauthorized" error
+
+    Example: granter has all msg's permissions
+      Given "acct1" has permission "LEVEL_ALL_MSGS"
+      When "acct1" attempts to grant "acct2" the permissions
+      """
+      { "level": "LEVEL_ALL_MSGS" }
+      """
+      Then expect an "unauthorized" error
+
+
+  Rule: limit_msg_types must be used with LEVEL_SOME_MSGS
+    Example: granting LEVEL_SOME_MSGS with limit_msg_types
+      Given "acct1" has no permissions
```

Comment:
shouldn't this be `Given "acct1" has permission "LEVEL_SUPER_ADMIN"`, expecting the result to be success
```suggestion
      Given "acct1" has no permissions
```